src/main/java/stsjorbsmod/powers/OldBookPower.java

```diff
@@ -0,0 +1,65 @@
+package stsjorbsmod.powers;
+
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import stsjorbsmod.actions.ExhumeCardsAction;
+import stsjorbsmod.util.CardMetaUtils;
+
+public class OldBookPower extends CustomJorbsModPower {
+    public static final StaticPowerInfo STATIC = StaticPowerInfo.Load(OldBookPower.class);
+    public static final String POWER_ID = STATIC.ID;
+
+    private AbstractCard card;
+
+    public OldBookPower(final AbstractCreature owner, AbstractCard card) {
```

Comment:
I'm not sure this is going to do the right thing if there are multiple copies of `Old Book` in the player's deck; it's probably best to have one stacking power rather than multiple distinct powers, since we'd only want one book to trigger at a time.